design/high-level-reactivity.md

```diff
@@ -234,12 +296,20 @@ The new system is designed to avoid this issue, and the implementation will be b
 
 After you analyze the two paths above, there should be a clear path to optimization through careful manual invalidations and disabling reactivity that will let you change a minimum of app code to "twist the knobs" on performance.
 
+// Oh Yeah! I'm seeing huge set of ideas. 
+// Let's implement this. So, we can build a great APM and make money :P
```

Comment:
Yes, anyone should be able to get the information out easily. Then it's a race to build a great experience on top of the raw data.